mlflow/projects/__init__.py

```diff
@@ -64,14 +64,14 @@ def _resolve_experiment_id(experiment_name=None, experiment_id=None):
 
     :param experiment_name: Name of experiment under which to launch the run.
     :param experiment_id: ID of experiment under which to launch the run.
-    :return: int
+    :return: str
```

Comment:
thanks! yeah experiment_id is meant to be string now. (this particular description: https://github.com/mlflow/mlflow/blob/80099417284f9c7377eb8af58b2845291abdea62/mlflow/tracking/fluent.py#L287 is wrong so it is confusing)